docs/pages/api-reference/connectors/kafka.md

```diff
@@ -44,9 +44,9 @@ SASL password.
 The name of the kafka topic that needs to be sourced into the dataset.
 </Expandable>
 
-<Expandable title="format" type='"json" | Avro' defaultVal="json">
-The format of the data in Kafka topic. Both `"json"` and 
-[Avro](/api-reference/connectors/avro) supported.
+<Expandable title="format" type='"json" | Avro | Protobuf' defaultVal="json">
+The format of the data in Kafka topic. `"json"`, 
+[Avro](/api-reference/connectors/avro) and [Protobuf](/api-reference/connectors/protobuf) supported.
```

Comment:
Does protobuf already work with Kinesis too or does it need a bit more work? If it already works, should we update similar documentation for Kinesis too?

Reply:
It doesn't work as of now. It needs a bit more work and tests. Post that we can add it to documentation